### PR TITLE
Fix HTML templates missing correct HTML tags

### LINKED ISSUE #14421: Email notification template used for matrix.org HS regressed
### Description

Sometime between 2022-10-31 and 2022-11-04, the matrix.org HS email notification template seemed to regress somehow (at least for my @jryans:matrix.org account).

Here are two example emails in each style for comparison:

## 2022-10-31

image

## 2022-11-04

image

I assume the 2022-10-31 style is still the intended one, as it has much nicer visual styling. Perhaps some configuration somewhere needs to be fixed...?

### Steps to reproduce

- Have an account on matrix.org
- Enable email notifications
- Cause an email to be sent

### Homeserver

matrix.org

### Synapse Version

1.71.0rc2 (b=matrix-org-hotfixes,39ea774cff,dirty)

### Installation Method

_No response_

### Platform

N/A

### Relevant log output

```shell
N/A
```

### Anything else that would be useful to know?

_No response_

### PR DESCRIPTION
**Pull request by erikjohnston**
Fixes #14421.

Broke in #13652.

@gsouquet would be useful if you could have a look to check that this makes sense?

### CODE REVIEW

File: synapse/res/templates/sso_partial_profile.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{% extends "_base.html" %}
```
Comment (clokep) [inline synapse/res/templates/sso_partial_profile.html:1]: This is incorrect -- this gets included onto other pages.

```suggestion
```

**Comment by DMRobertson** [comment]
Would be great if there was a dev script that rendered all the emails with dummy placeholders. Not sure how easy that is though.

**Comment by babolivier** [comment]
> Would be great if there was a dev script that rendered all the emails with dummy placeholders. Not sure how easy that is though.

I was thinking the same thing. Bonus points for uploading them to netlify to make them easy to check when a PR changes them.

**Review by clokep — APPROVED** [review]
Thanks!

Lint is failing, but looks unrelated.

**Comment by squahtx** [comment]
> Lint is failing, but looks unrelated.

Possibly due to #14128?

**Pull request merged**